shm/Player.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
```

Comment:
After #pragma once we should give empty spaces or it doesn't matter? After checking several student's PR I don't know.

Reply:
I would go with one new line

Reply:
Me too, but if I good remember Mateusz said that is just personal preferences.